src/main.rs

```diff
@@ -350,38 +352,44 @@ fn watch(
     loop {
         match rx.recv_timeout(Duration::from_secs(1)) {
             Ok(event) => match event {
-                DebouncedEvent::Create(b) | DebouncedEvent::Chmod(b) | DebouncedEvent::Write(b) => {
-                    if b.extension() == Some(OsStr::new("rs")) && b.exists() {
-                        let filepath = b.as_path().canonicalize().unwrap();
-                        let pending_exercises = exercises
-                            .iter()
-                            .find(|e| filepath.ends_with(&e.path))
-                            .into_iter()
-                            .chain(
+                Ok(events) => {
+                    for event in events {
+                        let b = event.path;
```

Comment:
Could you turn this into a more descriptive variable name like `event_path` or something? Thanks!

Reply:
That naming was adopted from the existing code, nevertheless I have updated the variable name as you suggested :)